samples/core/src/main/java/com/exadel/aem/toolkit/samples/models/WarriorDescriptionComponent.java

```diff
@@ -119,84 +121,84 @@
+    @DialogField(
+            label = "Birthday",
+            description = "Enter the birthday of your warrior"
+    )
     @DatePicker(
             type = DatePickerType.DATE,
             displayedFormat = "DD.MM.YYYY",
             valueFormat = "DD.MM.YYYY",
             maxDate = @DateTimeValue(day = 1, month = 12, year = 2019),
             typeHint = TypeHint.STRING
     )
-    @DialogField(
-            label = "Birthday",
-            description = "Enter birthday of you warrior"
-    )
+    @PlaceOnTab(WarriorDescriptionComponent.TAB_MAIN)
     @ValueMapValue
     private String birthday;
 
-    @DependsOnRef(name = "isLikesFruit")
-    @PlaceOnTab(WarriorDescriptionComponent.TAB_TASTES)
-    @DialogField(label = "Does your warrior like fruit?")
+    @DialogField(label = "Does your warrior like fruits?")
     @Checkbox
+    @DependsOnRef(name = "likesFruits")
+    @PlaceOnTab(WarriorDescriptionComponent.TAB_TASTES)
     @ValueMapValue
-    private boolean isLikesFruit;
+    private boolean likesFruits;
 
-    @PlaceOnTab(WarriorDescriptionComponent.TAB_FRUIT)
+    @DialogField(label = "Favorite fruit")
     @Autocomplete(
             multiple = true,
             forceSelection = true,
             datasource = @AutocompleteDatasource(namespaces = {"fruit"})
     )
-    @DialogField(label = "Favorite fruit")
+    @PlaceOnTab(WarriorDescriptionComponent.TAB_FRUITS)
     @ValueMapValue
     private String[] fruit;
 
-    @PlaceOnTab(WarriorDescriptionComponent.TAB_MAIN)
-    @RadioGroup(buttons = {
-            @RadioButton(text = "Funny", value = "always creepy smiles", checked = true),
-            @RadioButton(text = "Sad", value = "always steals your handkerchief"),
-            @RadioButton(text = "Angry", value = "always ignores you")
-    })
     @DialogField(
             label = "Character",
             description = "Choose character of your warrior",
             ranking = 4
     )
+    @RadioGroup(buttons = {
+            @RadioButton(text = "Funny", value = "always smile creepily", checked = true),
+            @RadioButton(text = "Sad", value = "always steal your handkerchief"),
+            @RadioButton(text = "Angry", value = "always ignore you")
+    })
+    @PlaceOnTab(WarriorDescriptionComponent.TAB_MAIN)
     @ValueMapValue
     private String character;
 
-    @DependsOnRef(name = "isLikesFilms")
-    @PlaceOnTab(WarriorDescriptionComponent.TAB_TASTES)
+    @DialogField(label = "Does your warrior like movies?")
     @Checkbox
-    @DialogField(label = "Does your warrior like films?")
+    @DependsOnRef(name = "likesMovies")
+    @PlaceOnTab(WarriorDescriptionComponent.TAB_TASTES)
     @ValueMapValue
-    private boolean isLikesFilms;
+    private boolean likesMovies;
 
+    @DialogField
     @Hidden
     @DependsOn(query = "'../../colorTheme'", action = DependsOnActions.FETCH)
     @DependsOnRef(name = "isDarkColorTheme", type = DependsOnRefTypes.BOOLSTRING)
     private String isDarkColorTheme;
```

Comment:
Pardon saying this) but `String isSomething` looks rather strange. If this is to merely hook some annotation on, make it boolean still

Reply:
done